src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
@forki as note, `Fake.DotNetSdk` is better as name (dotnet cli was the old name, in docs now is `.net core sdk`)